Monika After Story/game/script-topics.rpy

```diff
@@ -14435,3 +14435,61 @@ label monika_language_nuances:
     m 1tsu "So good day to you, [player].{w=0.3} {nw}"
     extend 1hub "Ahaha~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_architecture",
+            category=['misc'],
+            prompt="Architecture",
+            random=True
+        )
+    )
+
+label monika_architecture:
+    m 1esa "Hey, [player]! You know, I think there's this one major art branch that we've been missing to talk about all the time..."
+    m 3eub "Architecture!"
+    m 3esa "I have been reading a bit about it lately, so I find it quite interesting to discuss..."
+    m 1eua " If you come to think of it, architecture is one of the most common kinds of art in an everyday life."
+    m 1tub "Unless you live in a complete wasteland of some kind...{w=0.5}{nw}"
+    extend 1hub " Ahaha~"
+    m 1eka "But really, I am fascinated by how humanity tend to turn every craft into an art.{w=0.5}{nw}" 
+    extend 1eua " I think architecture is the greatest example of it."
+    m 1eub "Architecture can also tell a lot about the culture of a certain country it is located in: different monuments, statues, historical buildings, towers..."
```

Comment:
Not wrong per se, but using "tell a lot" like this without an indirect object seems a bit strange to me. I'd use "tell you a lot". Or maybe "say a lot".
"a certain country" is definitely weird in this context. Just use "the country". Either that or omit the "it is located in" part.